README.md

```diff
@@ -1,2 +1,20 @@
-# hw01
-Repository for homework 1
+# __hw01 Biography__
+
+## **Degrees**
+
+* PhD Comparative Human Development _2021_
+* MS Forensic Biology _2016_
+* BA Anthropology _2014_
+
+## **Research**
+
+1. I have two main projects right now
+  + I use eye tracking and verbal descriptions to understand how people make meaning from violence
+  + I used physiological and behavioral data in a haunted house to study how people play in scary situations
+
+2. You can find decriptions of these projects and others at [my website](https://sites.google.com/view/coltanscrivner/home)
+
+
+# __Report of Process__
+
+Since I had a guide to the syntax, this assignment seemed relatively straightforward. You simply create a fork on the HW repository, copy the repository from GitHub and clone it in RStudio on a new version control project. You then add the text to the README.md in R Markdown syntax. For the R Markdown syntax, I used [this website](https://guides.github.com/features/mastering-markdown/#examples). I did have to go back and check the course website to remind myself how to pull/push files between RStudio and GitHub. Once I saved the changed I made to README.md, I committed them, added a note, and pushed them to the repository on GitHub. The only issue I had was that I didn't create a fork at first, so I had to go back and create a new project under the forked repository. I also have a few revisions where I fixed typos. 
```

Comment:
This explanation would be best placed in the pull request comment. For in-line process explanations, I would suggest using the hashtag symbol at the beginning of the explanation in each line of code. This would avoid having the explanation show up along with your final product on the GitHub webpage.